include/access_management.js

```diff
@@ -131,14 +131,14 @@ module.exports = function(pb) {
     };
 
     /**
-     * 
+     *
      * @static
      * @method getRoleToDisplayNameMap
      * @param {Localization} ls
      * @return {Object}
      */
     SecurityService.getRoleToDisplayNameMap = function(ls) {
-        if (util.isFunction(ls)) {
+        if (util.isFunction(ls.get)) {
```

Comment:
This was returning false in my tests, so I moved it to the get function and it worked.